_footer.php

```diff
@@ -1,11 +1,11 @@
 </div>
 
-<div id="footer"><div style="float: left;"><a href="../legal.php">Legal</a></div>
-<a href="http://niwanetwork.org">Nintendo Independent Wiki Alliance</a> &copy; 2010-<?php echo(date('Y')); ?>
+<div id="footer"><div style="float: left;"><a href="/legal.php">Legal</a></div>
+<a href="/">Nintendo Independent Wiki Alliance</a> &copy; 2010-<?php echo(date('Y')); ?>
 </div>
 
 </div>
 
-<script type="text/javascript" src="https://apis.google.com/js/plusone.js"></script>
+<script src="https://apis.google.com/js/plusone.js"></script>
```

Comment:
Was this from Google+ ? If so it can definitely be killed

Reply:
Sounds like it. @prodigion can you remove this?

Reply:
Off-site discussion, will be done in different PR.